go/def.bzl

```diff
@@ -389,9 +390,24 @@ def emit_go_link_action(ctx, importmap, transitive_libs, cgo_deps, lib,
   # Avoided -s on OSX but but it requires dsymutil to be on $PATH.
   # TODO(yugui) Remove this workaround once rules_go stops supporting XCode 7.2
   # or earlier.
-  cmds += ["export PATH=$PATH:/usr/bin"]
   cmds += [
+    "export PATH=$PATH:/usr/bin",
     "export GOROOT=$(pwd)/" + ctx.file.go_tool.dirname + "/..",
+    "STAMP_XDEFS=()",
+  ]
+
+  stamp_input = []
+  if ctx.attr.stamp and ctx.attr.linkstamp:
+    stamp_input = [ctx.info_file, ctx.version_file]
```

Comment:
what are `ctx.info_file` and `ctx.version_file`? they don't seem to be documented on https://www.bazel.io/versions/master/docs/skylark/lib/ctx.html.